includes/class-amp-theme-support.php

```diff
@@ -331,6 +333,37 @@ public static function add_meta_viewport() {
 		echo '<meta name="viewport" content="width=device-width,minimum-scale=1">';
 	}
 
+	/**
+	 * Print analytics data.
+	 */
+	public static function add_analytics_data() {
+		$analytics_entries = apply_filters( 'amp_post_template_analytics', array() );
+
+		if ( empty( $analytics_entries ) ) {
+			return;
+		}
+
+		// Can enter multiple configs within backend.
+		foreach ( $analytics_entries as $id => $analytics_entry ) {
```

Comment:
Since the logic here is largely now a duplicate of `\amp_post_template_add_analytics_data()`, should a new function be added to `amp-helper-functions.php` called something like: `\amp_print_analytics()` which takes the `$analytics_entries` as its one arg? Then `\amp_post_template_add_analytics_data()` can be updated to call this common function.